src/KeyVault/KeyVault/Models/Key/PSKeyOperationResult.cs

```diff
@@ -13,34 +13,46 @@ public class PSKeyOperationResult
         // Summary: encryted result or wraped result is base64 format. decryted result or unwraped result is plain text
         public string Result { get; }
 
+        /// <summary>
+        /// For encrypt operation, this is the byte array format for the result of the encryption.
+        /// For decrypt operation, this is the byte array format for the decrypted data.
+        /// For wrap operation, this is the byte array of wrapped key.
+        /// For unwrap operation, this is the byte array of unwrapped key.
+        /// </summary>
+        public byte[] RawResult { get; }
```

Comment:
Why do we expose RawResult to public? It'll be difficult if we want to change it in the future.

Reply:
Make sense.